openstack/compute/v2/extensions/bootfromvolume/testing/fixtures.go

```diff
@@ -308,3 +308,53 @@ var NewVolumeTypeRequest = bootfromvolume.CreateOptsExt{
 		},
 	},
 }
+
+const ExpectedImageAndExistingVolumeWithTagRequest = `
+{
+	"server": {
+		"name": "createdserver",
+		"imageRef": "asdfasdfasdf",
+		"flavorRef": "performance1-1",
+		"block_device_mapping_v2":[
+			{
+				"boot_index": 0,
+				"delete_on_termination": true,
+				"destination_type":"local",
+				"source_type":"image",
+				"uuid":"asdfasdfasdf"
```

Comment:
nit: I'd use a real uuid here.

This is mostly stylistic, but I did actually get bitten by it once when the API added input validation and all the tests broke.

Reply:
while I agree with you, `asdfasdfasdf` is somehow used everywhere in Gophercloud :-(